bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/services/embed/OEmbedClientImplConfigurationFactory.java

```diff
@@ -39,6 +39,9 @@ public class OEmbedClientImplConfigurationFactory {
             description = "Configuration for defining oEmbed endpoints."
     )
     public @interface Config {
+
+        int defaultConnectionTimeout = 2000, defaultSocketTimeout=5000;
```

Comment:
These are constants by default, so the NAMING_CONVENTION_FOR_CONSTANTS ( :-) ) still applies.